test/e2e/specs/osba_app.go

```diff
@@ -0,0 +1,129 @@
+package specs
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/openshift/openshift-azure/pkg/util/ready"
+	"github.com/openshift/openshift-azure/test/e2e/standard"
+	"github.com/openshift/openshift-azure/test/sanity"
+)
+
+type osbaObj struct {
+	namespace string
+	data      []byte
+}
+
+var osbaApp, osbaService osbaObj
+
+func init() {
```

Comment:
don't think any of this really should be done in init(), nor are the globals needed